Blockzilla/OverlayView.swift

```diff
@@ -104,29 +104,33 @@ class OverlayView: UIView {
     func setSearchQuery(query: String, animated: Bool) {
         searchQuery = query
         let query = query.trimmingCharacters(in: .whitespaces)
-        
-        if false, let pasteBoard = UIPasteboard.general.string {
-            if let pasteURL = URL(string: pasteBoard), pasteURL.isWebPage() {
-                let attributedURL = NSAttributedString(string: pasteURL.absoluteString, attributes: [NSAttributedStringKey.foregroundColor: UIColor.white])
-                    copyButton.setAttributedTitle(attributedURL, for: .normal)
-            }
-            else {
-                setAttributedButtonTitle(phrase: pasteBoard, button: copyButton)
+
+        var showCopyButton = false
+
+        UIPasteboard.general.urlAsync() {
+            let handoffUrl = $0
```

Comment:
if you do `UIPasteboard.general.urlAsync() { handoffUrl in` I don't think we'd need this temporary variable